DelvUI/Helpers/ChatHelper.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace DelvUI.Helpers
+{
+    public class ChatHelper
+    {
+        #region Singleton
+        private ChatHelper()
+        {
+            var scanner = Plugin.PluginInterface.TargetModuleScanner;
```

Comment:
Everything we need to use from PluginInterface is now attached to Plugin directly;  could you use those instead?  It'll end up being less differences on the apiv4 branch.

```suggestion
            var scanner = Plugin.SigScanner;
```

Reply:
sure